### DISCUSSION TITLE
千方百计讨好开发者 - 论开源项目的开发流程 | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#10] [General]
# 千方百计讨好开发者 - 论开源项目的开发流程 | 高策

江湖小虾米

http://gaocegege.com/Blog/%E9%9A%8F%E7%AC%94/agile-in-open-source

**Comment by zhyon404** [comment]
Jeff Sutherland有[提到](https://www.quora.com/What-are-the-techniques-set-by-the-Scrum-guidelines-for-a-task-estimation-in-sprint-planning-Are-there-any-limitations-to-these-techniques)说可以不做task estimation

**Reply by gaocegege** [reply]
我去看看，谢谢分享
